### ISSUE TITLE
accesability Bug  no add block button is described

### ISSUE THREAD

**Issue by esmobg** [WordPress/gutenberg#6775]
**no add block button is described**
 bug is. 

no add block button is described
not readable with NVDA and Firefox browser

**To Reproduce**
1. Go to 'Gutemberg'
2. Click on Demo
3. Scroll down to 'Add block button'
4. no add block button is described
not readable with NVDA and Firefox browser

**Expected behavior**
read  and focus add botton and  activate with  keyboard 

**Desktop 
 - OS:  Windows 10 
 - Browser Firefox 
 - Version 59 

**Additional context**
Ghutemberg last  version

**Comment by danielbachhuber** [comment]
Thanks for the report, @esmobg.

Just for clarity, can you share a screenshot of the UI you're referring to?

**Comment by esmobg** [comment]
https://www.dropbox.com/s/pmel9fwd24lwa0k/Screenshot%202018-05-18%2011.52.27.png?dl=0
screenshot  on add botton

**Comment by esmobg** [comment]
https://www.dropbox.com/s/pmel9fwd24lwa0k/Screenshot%202018-05-18%2011.52.27.png?dl=0 
and back and forward  button not  active

**Comment by danielbachhuber** [comment]
@afercia Does this issue make sense to you?

**Comment by afercia** [comment]
@esmobg @danielbachhuber starting with the big shift with the "Quantum" version, Firefox has notably degraded performance with screen readers. It's a known issue and many NVDA users chose to use Firefox ESR (Extended Support Version) which is, I believe, Firefox 52. Mozilla has been working actively to improve the new Quantum performance together with the NVDA team and now Firefox 60 is almost as good as Firefox pre-Quantum.

So for now the recommendation is to test NVDA either with Firefox ESR or Firefox 60.

See also:
Firefox 60 and JAWS 2018 back in good browsing conditions together
https://www.marcozehe.de/2018/05/11/firefox-60-and-jaws-2018-back-in-good-browsing-conditions-together/

Firefox 59 becomes almost unusable on Windows 10 with NVDA
https://github.com/nvaccess/nvda/issues/8119

**Comment by afercia** [comment]
I've tested with Firefox 60 and NVDA 2018.1 and it works for me, as expected. The button is properly labeled with an `aria-label` attribute. Unfortunately there's nothing that can be done to improve Firefox 59 performance. 

Screenshot:

![screenshot 2](https://user-images.githubusercontent.com/1682452/40378146-fc216e66-5df2-11e8-971c-558d8e8bf02d.png)
